### PR TITLE
Minor text updates

### PR DESCRIPTION
**Pull request by michelle-rago**
## Changes proposed in this pull request:
- Updated text to improve readability

### CODE REVIEW

**Comment by michelle-rago** [comment]
@igorkorenfeld It's taking hours for these checks to pass

**Comment by igorkorenfeld** [comment]
Taking a look, I am not sure what might be causing it.

File: pages/domains/domains_benefits.md [hunk 1]
```diff
@@ -13,23 +13,23 @@ eleventyNavigation:
...
+Government organizations in the U.S. are eligible for .gov domains. This includes cities, towns, counties, election offices, states, tribal governments, courts, and more.
 
-Only U.S.-based government organizations are eligible for .gov domains. This includes any federal, state, local, or territorial government entity, or other publicly controlled entity. It also includes any tribal government recognized by the federal government or a state government. We use the [U.S. Census Bureau’s criteria for classifying governments](https://www.census.gov/programs-surveys/gus/technical-documentation/methodology/population-of-interest1.html){.usa-link--external} to help us determine eligibility.
+Federal, state, local, or territorial governments, or other publicly-controlled entities, are eligible. Tribal governments recognized by the federal government or a state government are eligible. We use the [U.S. Census Bureau’s criteria for classifying governments](https://www.census.gov/programs-surveys/gus/technical-documentation/methodology/population-of-interest1.html){.usa-link--external} to help us determine eligibility.
```
Comment (igorkorenfeld) [inline pages/domains/domains_benefits.md:32]: The "U.S.-based" portion does seem important to mention here

**Review by igorkorenfeld — APPROVED** [review]
Looks great! I just have one note.

File: pages/domains/domains_benefits.md (hunk 1 above)
Comment (igorkorenfeld) [reply pages/domains/domains_benefits.md:32]: Ah, ok cool that makes sense. That section is collapsed on the "files changed" so I didn't see that context. Thank you!

**Pull request merged**